src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -1876,6 +1887,9 @@ impl application::View for GraphEditor {
           // , (DoublePress , "!node_editing" , "space" , "double_press_visualization_visibility")
           , (Release     , "!node_editing" , "space" , "release_visualization_visibility")
           , (Press       , ""              , "cmd i" , "reload_visualization_registry")
+          , (Press       , ""              , "ctrl" , "enable_quick_visualization_preview")
+          , (Release     , ""              , "ctrl" , "disable_quick_visualization_preview")
```

Comment:
this does not work good on mac! On mac, `ctrl` is used only in special situations,l like killing a process. Use `cd` instead (which maps to ctrl on linu and windows).